app/(backroom)/me/me.module.css

```
@@ -0,0 +1,707 @@
+  text-transform: inherit;
+  letter-spacing: inherit;
+  transition: background .16s ease, color .16s ease;
+}
+
+.sectionNav button:hover {
+  color: var(--ink);
+  background: var(--section);
+}
+
+.sectionNav button[aria-pressed="true"] {
+  color: var(--ink);
```

Stylelint (17.14.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)